### PR TITLE
3.3 release prep

### PR DESCRIPTION
**Pull request by osfameron**
* merge 3.2 branch work
* update Start Using page
* bump server to 7.1
* Update compatibility

### CODE REVIEW

File: modules/devguide/examples/CloudConnect.java [hunk 1]
```diff
@@ -14,97 +14,82 @@
...
+  // Update these variables to point to your Couchbase Capella instance and credentials.
+  static String endpoint = "cb.njg8j7mwqnvwjqah.cloud.couchbase.com";
+  static String bucketname = "travel-sample";
+  static String username = "username";
+  static String password = "Password!123";
+  
+  public static void main(String... args) {
+    // tag::connect[]
+    // Configure TLS
+    SecurityConfig.Builder securityConfig = SecurityConfig
```
Comment (maria-robobug) [inline modules/devguide/examples/CloudConnect.java]: Do we still need this? Thought this was automatic now for Capella connections?

Comment (maria-robobug) [reply modules/devguide/examples/CloudConnect.java]: Works for me in Go without a Security config, that's why I ask. 
Not sure if the same or different for Java 😕 .

Comment (osfameron) [reply modules/devguide/examples/CloudConnect.java]: Don't know - I removed the Certificate wrangling, but I thought you still had to ensure it was TLS? @daschl to confirm?

Comment (osfameron) [reply modules/devguide/examples/CloudConnect.java]: Ah ok. Lemme try without, even better if so...

Comment (osfameron) [reply modules/devguide/examples/CloudConnect.java]: `An exception occured while executing the Java class. The DNS SRV record for 'cb.njg8j7mwqnvwjqah.cloud.couchbase.com' indicates TLS must be used when connecting to this cluster. Please enable TLS by setting the 'security.enableTLS' client setting to true. If the Cluster does not use a shared ClusterEnvironment, an alternative way to enable TLS is to prefix the connection string with "couchbases://" (note the final 's')`

Comment (osfameron) [reply modules/devguide/examples/CloudConnect.java]: Yup, works great with `couchbases://` in connection string...
That's definitely the way to go for Hello World example, thanks!

Comment (osfameron) [reply modules/devguide/examples/CloudConnect.java]: (also nice error message Michael & co!)

**Review by maria-robobug — APPROVED** [review]
👍

**Pull request merged**